src/main/kotlin/racingcar/domain/CarName.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain
+
+@JvmInline
+value class CarName(
+    val value: String,
+) {
+    init {
+        require(this.value.isNullOrBlank().not()) { "자동차 이름은 비어 있을 수 없습니다." }
```

Comment:
마이너하지만 `.not()` 을 사용하게 되면서 가독성이 약간은 떨어지게 된 거 같아요~
아래와 같이 좀 더 간결하게 작성하는 것도 좋을 거 같습니다!

```suggestion
        require(this.value.isNotBlank()) { "자동차 이름은 비어 있을 수 없습니다." }
```